cmd/cmds_ls.go

```
@@ -19,11 +20,7 @@ var cmdsLsCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		evt, err := getString(cmd.Flags(), "event")
-		if err != nil {
-			return err
-		}
-
+		evt := v.GetString("event")
```

error handling? silent error?